src/ex4/server/helper/pokemonHelper.js

```diff
@@ -0,0 +1,19 @@
+import { readFile } from './fileHandler.js'
+
+export function isPokemon(item) {
+    if (!isNaN(item)) return true
+    return false
+}
+
+export function isListOfPokemons(item) {
+    if (item.includes(",") && JSON.stringify(item.split(",")) === JSON.stringify(item.split(",").filter(item => !isNaN(item)))) {
```

Comment:
Please separate operations so the code will be more readable and maintainable
`if (item.includes(",") {`
`  const items = item.split(",");`
`  const filteredItems = item.split(",").filter;`
`  if (....) {`
`    return true`
`  }`
`}`

Reply:
Also this condition `JSON.stringify(item.split(",")) === JSON.stringify(item.split(",").filter(item => !isNaN(item)))` is an overkill of operations and logic.

`if (item.includes(",") && item.split(",").every(item => !isNaN(item))`
**or**
`if (item.includes(",") && !(item.split(",").some(item => isNaN(item)))`